guides/doc-Configuring_virt-who_VM_Subscriptions/master.adoc

```diff
@@ -0,0 +1,41 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: virt-who-vm-subs
+:virt-who-vm-subs:
+:ProductName: {ProjectServer}
+
+= {ConfiguringVMSubscriptionsDocTitle}
+
+ifdef::foreman-deb[]
+You cannot use the virt-who plugin because it requires the Katello plugin, which is not supported on Debian or Ubuntu.
+endif::[]
+ifdef::foreman-el[]
+You cannot use the virt-who plugin without the Katello plugin.
+endif::[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+ifdef::katello,satellite,orcharhino[]
+include::common/assembly_overview-of-vm-subscriptions.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-virt-who-kvm.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-virt-who-vmware.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-virt-who-nutanix.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-virt-who-hyperv.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-virt-who-openstack.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-virt-who-kubevirt.adoc[leveloffset=+1]
+
+include::common/modules/proc_editing-a-virt-who-configuration.adoc[leveloffset=+1]
+
+include::common/modules/proc_removing-a-virt-who-configuration.adoc[leveloffset=+1]
+
+include::common/assembly_troubleshooting-virt-who.adoc[leveloffset=+1]
+:!virt-who-vm-subs:
```

Comment:
```suggestion
```

suggestion: Not necessary because it's the end of the document; there's no more content to come that could use this macro.